### DISCUSSION TITLE
Get future event schedule

### DISCUSSION THREAD

**Discussion by timespacedecay** [br-g/openf1#9] [General]
Would be great to get future event schedules with session names start dates and times of each session.  A la http://ergast.com/api/f1/current/next.json (which will be dying at the end of this year).

**Comment by br-g** [comment]
Hello,
This is in the short term (early 2024) roadmap!

**Reply by botLuxas** [reply]
@br-g any idea on when in 2024 this will be? Looking very forward to have future data and stuff like race results, qualifying results after events have happened.

**Reply by br-g** [reply]
My current focus is optimizing cost and reliability of the current system, before adding new features.
In the short term, I recommend that you get this data from the Ergast API
